lib/fastlane/actions/deploygate.rb

```diff
@@ -72,14 +72,14 @@ def self.parse_response(response)
 
       def self.help_message(response)
         message =
-          case response.body['message']
+            case response.body['message']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.